Makefile

```diff
@@ -20,3 +37,70 @@ test:
 install:
 	go install -v ./pkg/... ./cmd/...
 
+.PHONY: clean
+clean:
+	rm -rf $(BUILD_DIR)
+
+.PHONY: build
+build: clean build-darwin-amd64 build-linux-amd64 build-windows-amd64
+
+.PHONY: build-darwin-amd64
+build-darwin-amd64: build-apiregister-gen-darwin-amd64 build-apiserver-boot-darwin-amd64 build-apiserver-builder-release-darwin-amd64
+
+.PHONY: build-apiregister-gen-darwin-amd64 build-apiserver-boot-darwin-amd64 build-apiserver-builder-release-darwin-amd64
+build-apiregister-gen-darwin-amd64 build-apiserver-boot-darwin-amd64 build-apiserver-builder-release-darwin-amd64: build-%-darwin-amd64:
```

Comment:
I think we will also want to build the vendored bins for the code generator, as well as include the vendored libraries.

apiserver-builder-release does these steps and produces a tar.gz, so maybe this just needs to run the release script and repackage the tar?

https://github.com/kubernetes-incubator/apiserver-builder/blob/master/cmd/apiserver-builder-release/main.go